bash/code_to_gast/bash_helpers.js

```diff
@@ -0,0 +1,59 @@
+const bash_router = require("./bash_router.js")
+
+/**
+ * Handles the script type which is the top level node that every command has
+ */
+function script_to_gast(node) {
+    gast = {"type": "root"}
```

Comment:
Would prefer if `gast` was fully declared with `const`.

Reply:
Fixed in next PR